node/cmd/guardiand/node.go

```diff
@@ -993,39 +998,47 @@ func runNode(cmd *cobra.Command, args []string) {
 	// Redirect ipfs logs to plain zap
 	ipfslog.SetPrimaryCore(logger.Core())
 
-	// If the wormchain sending info is configured, connect to it.
-	var wormchainKey cosmoscrypto.PrivKey
-	var wormchainConn *wormconn.ClientConn
-	if *wormchainURL != "" {
-		if *wormchainKeyPath == "" {
-			logger.Fatal("if wormchainURL is specified, wormchainKeyPath is required")
+	var accountantWormchainConn *wormconn.ClientConn
+	if *accountantContract != "" {
+		// TODO: wormchainKeyPath and wormchainKeyPassPhrase are being replaced by accountantKeyPath and accountantKeyPassPhrase.
```

Comment:
Does anything bad happen if both the accountant and wormchain parameters are populated?

Reply:
I guess I can add a check to block the wormchain ones if the accountant ones are populated.